lib/components/pop-up/index.tsx

```diff
@@ -15,6 +15,7 @@ interface PopUpProps {
   confirmText?: string;
   onConfirm: () => void;
   style?: React.CSSProperties;
+  size?: string;
```

Comment:
nit: you can have an enum or string union type here instead of a string ie: size?: "lg" | "sm" | "md"